packages/opentelemetry-plugin-http/src/utils.ts

```diff
@@ -434,5 +446,6 @@ export const getIncomingRequestAttributesOnResponse = (
     [AttributeNames.NET_PEER_PORT]: remotePort,
     [AttributeNames.HTTP_STATUS_CODE]: statusCode,
     [AttributeNames.HTTP_STATUS_TEXT]: (statusMessage || '').toUpperCase(),
+    [AttributeNames.HTTP_ROUTE]: route,
```

Comment:
I believe that if the route is undefined, it will appear in the exporter UI as `"http.route": "undefined"`